GameOfLife/game.cpp

```diff
@@ -1,38 +1,77 @@
 #include "Game.h"
 
-Board board(20,50);
+Board * board;
 
 Game::Game()
 {
-counterOfIterations = 0;
-filler.fillBoard(board);
-startInfiniteLoop();
+
+    counterOfIterations = 0;
+    int yourChoice;
+    do{
+
+        std::cout << "What you want to do:\n";
+        std::cout << "1. Load board form file txt\n";
+        std::cout << "2. Load board form file image\n";
```

Comment:
This is temporary menu, just to attach new feature to the project. Should we divide it into functions?